### ISSUE TITLE
Enhance model preparer support

### ISSUE THREAD

**Issue by quic-aishgupt** [quic/aimet#1998]
Adding the following support:

1. Support for following ops: floordivide, chunk, norm, avgpool2d, batchnorm, and groupnorm in model preparer.
2. Add transpose and reshape_as as math invariant ops
3. Modules with args and kwargs are now initialized in elementwise_ops using a common custom metaclass.
4. Added test cases for chunk and batch_norm in test_model_preparer.
